src/include/misc.h

```diff
@@ -333,19 +333,33 @@
 #define WT_STREQ(s, cs) (sizeof(cs) == 2 ? (s)[0] == (cs)[0] && (s)[1] == '\0' : strcmp(s, cs) == 0)
 
 /*
- * Check if a non-empty string matches a byte sequence of len bytes. The macro expects both strings
- * to be of non-zero length in order to be a match. The bytes argument does not need to be
- * null-terminated. Similar to other string related functions, this is not expected to work on NULL
- * values.
+ * Check if a literal string matches the length and content of the supplied bytes/len pair. The
+ * bytes argument does not need to be null-terminated, and it may be null if the supplied length is
+ * zero. Note this is different from the standard strncmp function, which with a supplied zero
```

Comment:
This is a good comment. I find the last sentence a bit hard to parse. Maybe something like this:
 Note this macro works differently than the standard strncmp function. When strncmp is supplied a
 zero length it returns true. When this macro gets a zero length it returns false.

Reply:
Thanks, I'll use your wording!